modules/test/playwright/pages/object-web/object-fields/ObjectFieldsFDSPage.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+export class ObjectFieldsFDSPage {
```

Comment:
Hey @danvb112, what do you think about remove the FDS info from the name of the page? Object fields already bring the idea of the list, since is in plural